powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1

```diff
@@ -1652,8 +1652,15 @@ function SetManagedEnvironmentSolutionCheckerEnforcementLevel
         return
     }
     
-    $governanceConfiguration.settings.extendedSettings.solutionCheckerMode = $Level
-
+    if($governanceConfiguration.settings.extendedSettings.solutionCheckerMode -eq $null)
```

Comment:
All these if/else statements can be simplified to a single line like this:

$governanceConfiguration.settings.extendedSettings | Add-Member -MemberType NoteProperty -Name 'solutionCheckerMode' -Value $Level **-Force**